### PR TITLE
Update pstn-usage-report.md, document data retention for call records

### PR DESCRIPTION
**Pull request by radoslag**

### CODE REVIEW

File: Teams/teams-analytics-and-reports/pstn-usage-report.md [hunk 1]
```diff
@@ -106,6 +107,33 @@ The following explains the different scenarios, and when Shared Correlation ID i
...
+| NO – Norway | 150 | 150 |
+| SE – Sweden | 180 | 180 |
+| UK – United Kingdom | 450 | 450 |
+| All other countries | 365 | 150 |
+
+Note that these rules apply to the PSTN and Direct Routing reports in Tenant Admin Center, data export, and the PSTN and Direct Routing call logs available via Microsoft Graph. Data retention in other reports, such as Call Analytics and Call Quality Dashboard, may differ.
+
+The retention period is based on the user's location (country). If the tenant administrator does not set a user's location, the default value for the location of a user is the location of the tenant.
+
+Tenant administrators have possibility to modify data retention period for the call records of their organization, by creating a support request to Microsoft.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/pstn-usage-report.md:135]:
```suggestion
Tenant administrators could modify the data retention period for the call records of their organization by creating a support request to Microsoft.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @radoslag, kindly commit the suggested changes. Many thanks.

File: Teams/teams-analytics-and-reports/pstn-usage-report.md (hunk 1 above)
Comment (SanderSade) [reply Teams/teams-analytics-and-reports/pstn-usage-report.md:135]: Can we please remove this sentence completely? This is not a functionality we want to publicize, especially as this comes with very heavy legal and technical limitations.

[the remark this answers is not in the document]
Comment (radoslag) [reply Teams/teams-analytics-and-reports/pstn-usage-report.md]: Removed the note about custom tenant-specified data retention.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
Hello @scanum, can you please help to get this PR approved and merged? Many thanks.

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**